### PR TITLE
ECS tasks can be manually run

### PR DESCRIPTION
**Pull request by DMcCallion**
Still need to setup schedule and lambda

### CODE REVIEW

File: setup/main.tf [hunk 1]
```diff
@@ -42,21 +60,183 @@ resource "aws_security_group" "steampulse_rds_sg" {
   }
 
   tags = {
-    Name = "allow_tls"
+
+    Name = "steampulse_rds_sg"
   }
 }
 
-resource "aws_db_instance" "steampulse_database" {
```
Comment (DMcCallion) [inline setup/main.tf:49]: This isn't actually a change, it just got moved around

File: setup/main.tf [hunk 2]
```diff
@@ -42,21 +60,183 @@ resource "aws_security_group" "steampulse_rds_sg" {
...
+  }
+
+
+  tags = {
+    Name = "steampulse_pipeline_ecs_sg"
+  }
+}
+
+
+#unfinished below
```
Comment (DMcCallion) [inline setup/main.tf:94]: The below resources that aren't commented out are working but I'm not 100% sure they have everything they need

Comment (gawsalya) [reply setup/main.tf:94]: It looks valid, I'm surprised at how the iam execution role actually needs so many policies -- the only way I would say you could cross check is by manually making and comparing them. For example with the task definition as inline policy -- I thought this is just declared with the aws_ecs_service resource as task definition as opposed to being said in the IAM role? If it works it works though

**Review by gawsalya — APPROVED** [review]
if it works! It works - just left a comment for thought but no suggested changes

**Review by rhKashif — APPROVED** [review]

**Pull request merged**